pkg/test/fit.go

```diff
@@ -403,7 +403,7 @@ func TestBIOSACMSizeCorrect() (bool, error) {
 		return false, err
 	}
 
-	if acm.Header.HeaderLen%32 != 0 {
+	if (acm.Header.HeaderLen*4)%4 == 0 {
```

Comment:
condition is always true.
check is wrong, it now always returns an error.
Change is unrelated to this PR.